servicetalk-concurrent-test-internal/src/main/java/io/servicetalk/concurrent/test/internal/AwaitUtils.java

```diff
@@ -122,12 +122,35 @@ public static boolean awaitUninterruptibly(CountDownLatch latch, long timeout, T
         }
     }
 
+    /**
+     * {@link BlockingQueue#take()} from the queue or throws unchecked exception in the case
+     * of InterruptedException.
+     * @param queue The queue to take from.
+     * @param <T> The types of objects in the queue.
+     * @return see {@link BlockingQueue#take()}.
+     */
+    @edu.umd.cs.findbugs.annotations.SuppressFBWarnings(
```

Comment:
After the suggestion above, this won't be necessary. FYI for future:

Instead of annotations, we use `main-exclusions.xml` files in `gradle` subfolder to suppress SpotBugs warnings we consider acceptable. See example: https://github.com/apple/servicetalk/blob/main/servicetalk-grpc-api/gradle/spotbugs/main-exclusions.xml